### PR TITLE
Dockerfile for Deployment

### PR DESCRIPTION
**Pull request by expedited**
Addresses https://github.com/BernardZhao/ocfstatic/issues/122

Changes: 
* Added a 2 stage Dockerfile to build the package, and then copy to nginx Docker image 

* Added Makefile to push docker image

### CODE REVIEW

**Comment by nikhiljha** [comment]
```
> cwebp-bin@5.1.0 postinstall /node_modules/cwebp-bin
> node lib/install.js

  ⚠ spawn /node_modules/cwebp-bin/vendor/cwebp ENOENT
  ⚠ cwebp pre-build test failed
  ℹ compiling from source
  ✖ Error: Command failed: /bin/sh -c ./configure --disable-shared --prefix="/node_modules/cwebp-bin/vendor" --bindir="/node_modules/cwebp-bin/vendor"
configure: error: in `/node_modules/cwebp-bin/3ecb56d3-4d59-46f0-b6d1-75e64afb73d8':
configure: error: no acceptable C compiler found in $PATH
See `config.log' for more details
```

You might need to `apk add gcc g++` in the build Dockerfile.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# build stage
+FROM node:13-alpine AS build-env
+COPY . .
```
Comment (nikhiljha) [inline Dockerfile:3]: RUN apk add gcc g++

**Comment by expedited** [comment]
So ran into some more issues here, not sure how to proceed with this one. Looks like there's a dependency called cwebp-bin that's being installed and it doesn't work on Alpine because it's compiled with glibc. So alternative would be to try another distro but not sure what that would break @BernardZhao @nikhiljha 
https://i.imgur.com/fFiIhrP.png

Also unsure about this final error I get here
https://i.imgur.com/MnD42Za.png

**Comment by nikhiljha** [comment]
@expedited Did you `rm -rf node_modules` in the build step before `npm install`? I think you added that yesterday but you haven't pushed the new changes so I can't see.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# build stage
+FROM node:13-slim AS build-env
+COPY . .
+RUN rm -rf node_modules
```
Comment (BernardZhao) [inline Dockerfile:4]: I think instead of doing this, just add a `.dockerignore` file with node_modules in it.

**Comment by BernardZhao** [comment]
The deploy issues are just an artifact issues since I upgraded node

**Pull request merged**